src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -105,13 +135,13 @@ public void start(Stage primaryStage) throws Exception {
         // menuBar.getMenus().addAll(menuConvo);
 
         ObservableList<String> usersList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
         ListView<String> users = new ListView<String>(usersList);
         ObservableList<String> convoList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
         ListView<String> conversations = new ListView<String>(convoList);
         ObservableList<String> messageList = FXCollections.observableArrayList(
-                 "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
+                "Julia", "Ian", "Sue", "Matthew", "Hannah", "Stephan", "Denise");
         ListView<String> messages = new ListView<String>(messageList);
 
         // userTf.setStyle("-fx-base: #b6e7c9;");
```

Comment:
I'm commenting here since GitHub won't let me comment on unchanged sections. The selection directly below contains a couple calls to `setMaxWidth(Double.MAX_VALUE)`. I've never really used JavaFX so I might be missing something, but I wonder why you need this? Is the default max width too small?